src/Marten/Schema/UniqueIndexType.cs

```diff
@@ -3,6 +3,7 @@
     public enum UniqueIndexType
     {
         DuplicatedField,
-        Computed
+        Computed,
+        Tenanted

```

Comment:
I think that this is redundant. I don't see much value on that, especially that. Unique Index might be both computed and tenanted.